x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -302,20 +302,6 @@ public void testKibanaUserRole() {
         assertThat(kibanaUserRole.indices().allowedIndicesMatcher("indices:foo")
                 .test(randomAlphaOfLengthBetween(8, 24)), is(false));
 
-        Arrays.asList(".kibana", ".kibana-devnull").forEach((index) -> {
```

Comment:
I removed the tests that ensure we have access for these indices, as it felt wrong to be writing tests to make sure privileges weren't granted to indices, but if you'd prefer I leave the tests and ensure there is no access, just lemme know.